Weather Man/Weather_Man/FileReader.py

```diff
@@ -0,0 +1,59 @@
+class FileReader():
+    def __init__(self):
+        self.pkt = []
+        self.max_temperature_c = []
+        self.mean_temperature_c = []
+        self.min_temperature_c = []
+        self.dew_point_c = []
+        self.mean_dewpoint_c = []
+        self.min_dewpoint_c = []
+        self.max_humidity = []
+        self.mean_humidity = []
+        self.min_humidity = []
+        self.max_sea_level_pressureh_pa = []
+        self.mean_sea_level_pressureh_pa = []
+        self.min_sea_level_pressureh_pa = []
+        self.max_visibility_km = []
+        self.mean_visibility_km = []
+        self.min_visibility_km = []
+        self.max_wind_speed_km_h = []
+        self.mean_wind_speed_km_h = []
+        self.max_gust_speed_km_h = []
+        self.precipitation_mm = []
+        self.cloud_cover = []
+        self.events = []
+        self.wind_dir_degrees = []
+
+    def data_filler(self, file_path_list=[]):
+        i = 0
+        while(i < len(file_path_list)):
+            with open(file_path_list[i]) as read_file:
+                next(read_file)
+                for line in read_file:
+                    one_day_data = line.split(",")
+
+                    self.pkt.append(one_day_data[0])
```

Comment:
Wouldn't it make more sense to store related data together? How would you know in this case the temperature at a certain index belongs to which day?

Make dataclass that stores data for each row and perform calculations based on that.

Reply:
Actually sir all its monthly/yearly data
if I want data for 2005 aug month data
all the data will be stored in 23 lists and index will match the day from 0 to 30 
0 index will have the data of first day of aug 2005